metaflow/plugins/conda/conda_step_decorator.py

```diff
@@ -79,8 +84,11 @@ def _lib_deps(self):
         base_deps = self.base_attributes['libraries']
         deps.update(base_deps)
         step_deps = self.attributes['libraries']
-        if isinstance(step_deps, collections.Mapping):
-            deps.update(step_deps)
+        if isinstance(step_deps, (unicode, basestring)):
+            step_deps = step_deps.strip('"{}"')
+            if step_deps:
+                step_deps = dict(a.split(':') for a in step_deps.split(','))
```

Comment:
Similarly, to deal with spaces and whatnot, I suggest something like this:
`dict(map(lambda x: x.strip().strip('"\''), a.split(':')) for a in step_deps.split(','))

Reply:
Done.